connector/MongoDBAtlasODBC.pq

```diff
@@ -1,23 +1,25 @@
 [Version = "0.0.0"]
 section MongoDBAtlasODBC;
 
-// When set to true, additional trace information will be written out to the User log. 
-// This should be set to false before release. Tracing is done through a call to 
-// Diagnostics.LogValue(). When EnableTraceOutput is set to false, the call becomes a 
+// When set to true, additional trace information will be written out to the User log.
+// This should be set to false before release. Tracing is done through a call to
+// Diagnostics.LogValue(). When EnableTraceOutput is set to false, the call becomes a
 // no-op and simply returns the original value.
 EnableTraceOutput = false;
 
-AtlasSQLLink="https://www.mongodb.com/atlas/sql";
+AtlasSQLLink = "https://www.mongodb.com/atlas/sql";
 
 [DataSource.Kind = "MongoDBAtlasODBC", Publish = "MongoDBAtlasODBC.Publish"]
 shared MongoDBAtlasODBC.Contents = Value.ReplaceType(
-    (mongoDbUri, database, optional options) => MongoDBAtlasODBCImpl(mongoDbUri, database, options),
+    (mongoDbUri, database, loglevel, optional options) =>

```

Comment:
I think we should check with Alexi on whether to make loglevel optional.

I had the same question when I was working on setting the transform mode from JSON or Text.
Because the way it is here the user has to choose the value in the dropdown.  The other method is to say "INFO" is default in the FieldCaption or FieldDescription and keep this optional.  

It's too bad there doesn't seem to be a way to have the drop-down already prepopulated with one of the values.  With it optional we can add the check to